documentation/man/features/kw-explore.rst

```diff
@@ -45,5 +45,9 @@ OPTIONS
 -H | \--only-header:
   With this option, it is possible to show only the results from the header.
 
+-C<num> | \--show-context=<num>:
+  Show <num> lines of additional context above and below the matched line.
+  If <num> is not specified, the default value of 3 will be used.
```

Comment:
How about creating an example section and adding one or two examples of this feature?